src/ts/multiSend/batchTx.ts

```diff
@@ -0,0 +1,33 @@
+import { EncodedTx } from "../forwarders/forwarder";
+import { MultiSend } from "../../typedContracts/MultiSend";
+
+interface TxToBatch {
+  to: string;
+  data: string;
+  revertIfFail: boolean;
+}
+export class MultiSender {
+  constructor(private readonly multiSend: MultiSend) {}
```

Comment:
You only need a MultisendInterface in here, not a full contract. It's probably better to replace this with the address so that you can handle all the multisend interface stuff internally
```
private readonly multiSendInterface;
constructor(private readonly multiSendAddress: string) {
      this.multiSendInterface = new MultiSendInterface()
}
```